Python/Product/TestAdapter/TestAdapter.csproj

```diff
@@ -114,28 +117,18 @@
       <Link>Guids.cs</Link>
     </Compile>
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
     <Compile Include="RunFromContextFileExtensions.cs" />
     <Compile Include="..\..\..\Common\Product\TestAdapter\ServiceProviderExtension.cs">
       <Link>ServiceProviderExtension.cs</Link>
     </Compile>
-    <Compile Include="TestAnalyzer.cs" />
-    <Compile Include="..\..\..\Common\Product\TestAdapter\TestContainer.cs">
-      <Link>TestContainer.cs</Link>
-    </Compile>
-    <Compile Include="TestContainerDiscoverer.cs" />
     <Compile Include="TestDiscoverer.cs" />
     <Compile Include="TestExecutor.cs" />
-    <Compile Include="..\..\..\Common\Product\TestAdapter\TestFileAddRemoveListener.cs">
-      <Link>TestFileAddRemoveListener.cs</Link>
-    </Compile>
-    <Compile Include="..\..\..\Common\Product\TestAdapter\TestFilesUpdateWatcher.cs">
-      <Link>TestFilesUpdateWatcher.cs</Link>
-    </Compile>
-    <Compile Include="TestExecutorProjectContext.cs" />
-    <Compile Include="TestMethodResolver.cs" />
-    <Compile Include="..\..\..\Common\Product\TestAdapter\VsProjectExtensions.cs">
-      <Link>VsProjectExtensions.cs</Link>
-    </Compile>
```

Comment:
Or to substitute for a different discoverer/executor implementation, while still reusing whatever core Python functionality (analysis/coverage.py/etc.) can be reliably added into the core DLL. Or better yet, its own assemblies, so that we can keep startup time down by lazily loading components as much as possible.